README.md

```diff
@@ -68,6 +68,7 @@ You can access these pages on your computer using one of the following clients:
 - [Elixir client](https://github.com/edgurgel/tldr_elixir_client) (binaries not yet available)
 - [Emacs client](https://github.com/kuanyui/tldr.el), available on [MELPA](https://github.com/melpa/melpa)
 - [Go client](https://github.com/pranavraja/tldr): `go get github.com/pranavraja/tldr` (or [platform binaries](https://github.com/pranavraja/tldr/releases))
+- [Go pre-packaged](https://github.com/leighmcculloch/tldr): `go get 4d63.com/tldr` or `brew install leighmcculloch/tldr/tldr` or [binaries](https://github.com/leighmcculloch/tldr/releases)
```

Comment:
Lets keep that main link as [Go client] for consistency.

You can split the description into 2 lines. First line can be the `go get` one. And second can have the `brew install` and binaries link.

Reply:
I made the change you requested, but then realized the other languages that have multiple all group under a single one for each language. I just pushed another change to bring it inline with that, do you prefer the latest?